src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,370 @@
+  with same methods as BULK transfers with `transferIn(endpointNumber, length)`
+  and `transferOut(endpointNumber, data)`.
+- ISOCHRONOUS transfers, used for streams of data, like video and sound are
+  handled with `isochronousTransferIn(endpointNumber, packetLengths)` and
+ `isochronousTransferOut(endpointNumber, data, packetLengths)`.
+- BULK transfers, used to transfer a large amount of non-time-sensitive data in
+  a reliable way are handled with `transferIn(endpointNumber, length)` and
+ `transferOut(endpointNumber, data)`.
+
+You may also want to have a look at Mike Tsao's [WebLight
+project](https://github.com/sowbug/weblight) which provides a ground-up example
+of building a USB-controlled LED device designed for the WebUSB API (not using
+an Arduino here). You'll find hardware, software, and firmware.
+
+## Tips
+
+Debugging USB in Chrome is made easier with the internal page
+`chrome://device-log` where you can see all USB device related events in one
+single place.
+
+<img style="width:723px; max-height:414px" src="/web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-device-log-page.png" alt="Internal page to debug Web USB"/>
+
+Early adopters who want to test their existing devices with WebUSB before they
+can update their firmware or the [Public Device Registry](https://wicg.github.io/webusb/#public-device-registry) 
+is [implemented](https://bugs.chromium.org/p/chromium/issues/detail?id=598765) 
+are not out ouf luck there thanks to a dedicated switch. To disable checking of
+the WebUSB allowed origins descriptors that implement a CORS-like mechanism to
+secure origin to device communications, run `chrome` with the
+`--disable-webusb-security` [switch](https://www.chromium.org/developers/how-tos/run-chromium-with-flags).
+
+On most Linux systems, USB devices are mapped with read-only permissions by
+default. To allow Chrome to open a USB device, you will need to add a new [udev rule](https://www.freedesktop.org/software/systemd/man/udev.html).
+Create a file at `/etc/udev/rules.d/50-yourdevicename.rules` with the following
+content:
+
+```
+SUBSYSTEM=="usb", ATTR{idVendor}=="[yourdevicevendor]", MODE="0664", GROUP="plugdev"
+```
+
+where `[yourdevicevendor]` is `2341` if your device is an Arduino for instance.
+`ATTR{idProduct}` can also be added for a more specific rule. Make sure your
+`user` is a member of the `plugdev` group.  Then, just reconnect your device.
+
+## What's next
+
+A second iteration on the WebUSB API will look at [Shared Worker](https://developer.mozilla.org/fr/docs/Web/API/SharedWorker)
+and [Service Worker](https://jakearchibald.github.io/isserviceworkerready/resources.html)
+support. Imagine for instance a security key website using the WebUSB API that
+would install a service worker to act as a middle man to authenticate users.
+
+And for your greatest pleasure, the WebUSB API will come later to Android.
+
+## Resources
+
```

Comment:
Done.

Not sure about best practice to file bugs though. 
@reillyeon may help there.